<launch>
  <node pkg="gige_camera_baumer17"
	type="run_gige_capture_node.sh"
	name="gige_camera_baumer17">
    <param name="calib_file"
	   value="package://gige_camera_baumer17/calib/sn0617531815_20150909.yml" />
  </node>

  <node pkg="nodelet" type="nodelet"
	name="image_proc_manager" args="manager" output="screen" />
  <node pkg="nodelet" type="nodelet" name="image_crop"
	args="load image_proc/crop_decimate image_proc_manager">
    <param name="x_offset" type="int" value="4" />
    <param name="y_offset" type="int" value="0" />
    <param name="width" type="int" value="1280" />
    <param name="height" type="int" value="960" />
    <param name="camera_name" value="image_crop"/>
    <remap from="camera/image_raw"
	   to="/gige_camera_baumer17/image_raw" />
    <remap from="camera/camera_info"
	   to="/gige_camera_baumer17/camera_info" />
    <remap from="camera_out/camera_info"
	   to="/null/camera_info" />
  </node>

  <node pkg="tomato_handeye"
	type="camera_info_publisher.py"
	name="camera_info_publisher" />

  <node pkg="nodelet" type="nodelet" name="image_rectify"
	args="load image_proc/rectify image_proc_manager">
    <remap from="image_mono"
	   to="/camera_out/image_raw" />
    <remap from="camera_info"
	   to="/camera_out/camera_info" />
    <remap from="image_rect"
	   to="/camera_out/image_rect" />
  </node>

</launch>
